client/src/app/pages/applications/components/application-list-expanded-area/application-list-expanded-area-extras.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { TableComposable, Thead, IRow, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+
+import {
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+} from "@patternfly/react-core";
+
+import { Application, Task } from "@app/api/models";
+import { ApplicationTags } from "../application-tags";
+import { ApplicationAnalysisStatus } from "../../components/application-analysis-status";
+
+export interface IApplicationListExpandedAreaProps {
+  application: Application;
+  tasks: Task[];
+}
+
+export const ApplicationListExpandedAreaExtras: React.FC<
```

Comment:
Maybe we can change the file/component name(s) to "Addons" instead of "Extras"? WDYT?